packages/client/src/helpers/FetchGraphQLClient.ts

```diff
@@ -0,0 +1,11 @@
+import { GraphQLClient } from 'graphql-request';
+import { PatchedRequestInit } from 'graphql-request/dist/types';
```

Comment:
could that be possible to import types from `graphql-request` (not from `dist`)?

If the params are not exported we could try to get one on our own

```
type ClientOptions = ConstructorParameters<GraphQLClient>[1];
type Options = Omit<ClientOptions, "fetch">;
```